app/database/queries/breeze_buddy/template.py

```
@@ -135,25 +131,21 @@ def get_templates_list_query(filters: Dict[str, Any]) -> Tuple[str, List[Any]]:
     conditions = []
     values = []
 
-    # Handle reseller filtering (supports both single and multiple) with COALESCE for backward compatibility
+    # Handle reseller filtering (supports both single and multiple)
     if "reseller_ids" in filters and filters["reseller_ids"]:
```

Ruff (0.15.5)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)